test/Tizen.NUI.Seamless/NUISimpleCaller/NUISimpleCaller.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.Applications;
+
+namespace NUISimpleCaller
+{
+    class Program : NUIApplication
+    {
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+            Initialize();
+        }
+
+        void Initialize()
+        {
+            Window.Instance.KeyEvent += OnKeyEvent;
+            Window.Instance.TouchEvent += OnTouchEvent;
+
+            TextLabel text = new TextLabel("NUI Simple Caller Sample");
+            text.HorizontalAlignment = HorizontalAlignment.Center;
+            text.VerticalAlignment = VerticalAlignment.Center;
+            text.TextColor = Color.Blue;
+            text.PointSize = 12.0f;
+            text.HeightResizePolicy = ResizePolicyType.FillToParent;
+            text.WidthResizePolicy = ResizePolicyType.FillToParent;
+            Window.Instance.GetDefaultLayer().Add(text);
+
+
+            TransitionOptions = new TransitionOptions(GetDefaultWindow());
```

Comment:
could you explain what kind of transition effect would be shown?
can it be changed?

Reply:
I think animation type design needs consideration.
Currently, SlideIn / SlideOut animation is implemented as a basic animation. (line 33~34)
I am thinking more about Fade as a basic animation,
The user can inherit TransitionAnimation and implement an animated animation.
I think it takes a lot of time to design the TransitionAnimation class. I think this class should be available not only app transition but also overall use such as view / layout / xaml.